lib/components/AuthorizedForFirstComponent.react.js

```diff
@@ -0,0 +1,29 @@
+import RoleAwareComponent from './RoleAwareComponent.react';
+
+export default class AuthorizedForFirst extends RoleAwareComponent {
+
+    constructor(props) {
+        super(props);
+
+        this.allowedRoles = [];
+        // roles of the user - usually fetched during authentication
+        this.userRoles = undefined;
```

Comment:
userRoles are `undefined` - this won't work... `this.rolesMatched()` will always throw exception...